api/src/sockets/kots/DeploySocket.ts

```diff
@@ -133,6 +132,26 @@ export class KotsDeploySocketService {
               // create the Restore resource
               await velero.restore(snapshotName, app.restoreInProgressName);
               logger.info(`Created Restore object ${app.restoreInProgressName}`);
+            } else {
+              const restorePhase = _.get(restore, "status.phase");
+              if (restorePhase === Phase.Completed) {
+                // Switch operator back to deploy mode on the restored sequence
+                const backup = await velero.readBackup(restore.spec.backupName);
+                if (backup.metadata.annotations) {
```

Comment:
I think it would be better to throw here, 142, and 145 so there is something logged if the annotations are missing

Reply:
how about logging rather than restore? i would be concerned that if i threw an exception in these socket loops then it would affect other apps

Reply:
logging's good, I was thinking it would log in 158